### ISSUE TITLE
Ffmpeg error

### ISSUE THREAD

**Issue by nefty1029** [n00mkrad/cupscale#3]
Hello, this is my first time using this program.
I just installed all the dependencies using the compatibility checker, 
yet when I load a video, the following error happens:

Failed to open image:
FailedToExecuteCommand `"ffmpeg.exe" -nostdin -v -1 -i "C:/Users/Nefty/AppData/Local/Temp/magick-YVblTv84OTD1qa-w7268u-SqoW99IIzf" -vframes 1 -vcodec pam -an -f rawvideo -y "C:/Users/Nefty/AppData/Local/Temp/magick-qfjAzj7plzwz2-zJjbWFvDLoC_mGRVbw.pam"' (-1) @ error/delegate.c/ExternalDelegateCommand/478

Stack Trace:
   at ImageMagick.NativeInstance.CheckException(IntPtr exception, IntPtr result)
   at ImageMagick.MagickImage.NativeMagickImage.ReadFile(IMagickSettings`1 settings)
   at ImageMagick.MagickImage.Read(String fileName, IMagickReadSettings`1 readSettings, Boolean ping)
   at ImageMagick.MagickImage..ctor(String fileName)
   at Cupscale.ImageUtils.ImgUtils.GetMagickImage(String path, Boolean allowTgaFlip)
   at Cupscale.UI.PreviewUI.DroppedImageIsValid(String path)

I'm not sure where I went wrong. Any advice is appreciated.

**Comment by n00mkrad** [comment]
Make sure you select the video tab first (at the top of the window), otherwise it will try to load the video into the image preview, which seems to fail if ffmpeg is not installed system-wide.

**Comment by nefty1029** [comment]
Whoops, well that works.
Thank you for your help.

**Comment by nefty1029** [comment]
Sorry to bother you again but the upscaling finished but then it got stuck in the cmd. 
I checked the log and received the below problem:

[IOUtils] Copying directory "C:\Users\nefty\AppData\Roaming\Cupscale\ShippedEsrgan\utils" to "C:\Users\nefty\AppData\Roaming\Cupscale\ShippedEsrgan\py\utils" (Move: False - RemoveFromName: )

I know that ffmpeg is supposed to convert the frames to videos but I'm not sure what to do.  
I tried entering some commands (I thought the program wants me to type the output folder) 
but I want to make sure that I do not mess up the program.
![Capture2](https://user-images.githubusercontent.com/19631520/100216077-cf7c9380-2f4c-11eb-984c-9b7320ea2023.PNG)
